projects/iio_demo/src/main.c

```diff
@@ -157,9 +198,17 @@ int main(void)
 	};
 #endif // XILINX_PLATFORM
 
+#ifdef ADUCM_PLATFORM
+
+	/* Aducm platform dependent initialization for UART. */
+	struct aducm_uart_init_param platform_uart_init_par = {
+			.parity = UART_NO_PARITY,
+			.stop_bits = UART_ONE_STOPBIT,
+			.word_length = UART_WORDLEN_8BITS };
```

Comment:
nitpick: end bracket should be on new line.

Reply:
done